README.md

```diff
@@ -1,5 +1,7 @@
 # pulumi-hugo
 
+[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/pulumi/pulumi-hugo)
```

Comment:
Any reason not to put this under the VS Code devcontainer heading? (I'm not sure what the conventions are around this, but assuming this launches/uses the devcontainer, it'd seem to make sense to put it there.)

Reply:
I can relocate it there if you prefer. I was intentional in setting it at the top for convenient and obvious "oh i just do this to get started" but of course it is reasonable to nest it in the appropriate section.

Reply:
Resolved in [26e76ba](https://github.com/pulumi/pulumi-hugo/pull/3909/commits/26e76ba2785d17f0ff11942a25a01ee660ec2790)